site/public/js/grade-inquiry.js

```diff
@@ -1,6 +1,25 @@
 /* global buildCourseUrl, WebSocketClient */
 /* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
 
+function saveDraft() { 
+    //display the saved draft text on grade inquiry box
+    var draftContent = localStorage.getItem('content');
+    if(draftContent === null) {
+        draftContent = {};
+    }else{
+        draftContent = JSON.parse(draftContent);
+    }
+    var elements = document.getElementsByClassName('markdown-textarea fill-available ');
+    for (var i = 0; i < elements.length; i++) {
+        var element = elements[i];
+        var elementId = element.getAttribute('id');
+        const reply_text_area = $(element);
+        if(draftContent.hasOwnProperty(elementId)){
```

Comment:
fix formatting here

Reply:
Thank you for the feedback. I have fixed the formatting.